js/packages/teams-ai/src/authentication/Authentication.ts

```diff
@@ -0,0 +1,292 @@
+/* eslint-disable security/detect-object-injection */
```

Comment:
Per line instead of per file. For files that already have this, could you file an issue, and we can discuss in a meeting?